.travis.yml

```diff
@@ -11,20 +11,20 @@ branches:
 language: generic
 
 env:
-  - NPM_TAG=latest IMAGE_NAME=theia-full
-  - NPM_TAG=next IMAGE_NAME=theia-full
-  - NPM_TAG=latest IMAGE_NAME=theia
-  - NPM_TAG=next IMAGE_NAME=theia
-  - NPM_TAG=latest IMAGE_NAME=theia-java
-  - NPM_TAG=next IMAGE_NAME=theia-java
-  - NPM_TAG=latest IMAGE_NAME=theia-go
-  - NPM_TAG=next IMAGE_NAME=theia-go
-  - NPM_TAG=latest IMAGE_NAME=yangster
-  - NPM_TAG=next IMAGE_NAME=yangster
-  - NPM_TAG=latest IMAGE_NAME=theia-python
-  - NPM_TAG=next IMAGE_NAME=theia-python
-  - NPM_TAG=latest IMAGE_NAME=theia-swift
-  - NPM_TAG=next IMAGE_NAME=theia-swift
+  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=8
```

Comment:
Sorry, I was wrong: pinning was correct, since we use that version to fetch a specific tarball from nodejs.org, which fails when we only request version 8.

Reply:
ok, now we know the reason :D